ui-admin/src/portal/siteContent/AddPageModal.tsx

```diff
@@ -82,20 +85,37 @@ const AddPageModal = ({ portalEnv, portalShortcode, insertNewPage, onDismiss }:
             setPage({ ...page, title: event.target.value })
           }}/>
 
-        <label htmlFor="inputPagePath">Page Path</label>
-        <div className="input-group">
-          <div className="input-group-prepend">
-            <span className="input-group-text" style={{ borderTopRightRadius: 0, borderBottomRightRadius: 0 }}
-              id="pathPrefix">{portalUrl}/</span>
-          </div>
-          <input type="text"
-            className="form-control"
-            id="inputPagePath"
-            value={page.path} aria-describedby="pathPrefix"
-            data-testid="page-path-input"
-            onChange={event => {
-              setPage({ ...page, path: event.target.value })
+        <>
+          <label htmlFor="inputPagePath">Page Path</label>
+          <InfoPopup title="Page Path" content={
+            <div>
+                The path to the page within your portal. For example, a path of&nbsp;
+              <code>my-path</code> will be available at the URL:&nbsp;
+              <br/><br/>
+              <code>{portalUrl}/my-path</code>
+            </div>
+          }/>
+        </>
+        <input
+          className="form-control"
+          id="inputPagePath"
+          value={page.path}
+          data-testid="page-path-input"
+          onChange={event => {
+            setPage({ ...page, path: event.target.value })
+          }}/>
+
+        <div className="d-flex mt-2">
+          <Checkbox
+            label={'Hide Navbar'}
+            checked={page.hideNavbar}
+            onChange={checked => {
+              setPage({ ...page, hideNavbar: checked })
             }}/>
+          <InfoPopup title="Page Path" content={
+            `If checked, navbar content will be hidden on this page and only the logo,
```

Comment:
kinda wondering now...should we also hide footer navigation too?

Reply:
great question -- I suspect no, since the footer has legally required stuff.  and mainly we just want to avoid them hitting "register"